src/content.ts

```diff
@@ -54,10 +61,13 @@ async function handle_baheth_content(
 
   // render transcription download button if possible
   if (
-    baheth_data?.transcription_txt_link ||
-    baheth_data?.transcription_pdf_link ||
-    baheth_data?.transcription_srt_link ||
-    baheth_data?.transcription_epub_link
+    page_type === 'video' &&
```

Comment:
Shouldn't we decouple the download button and the transcription section rendering? Even if we don't have the download links, if the transcription is available, then we should render it, right?